deploy/helm/elastic-agent/examples/eck/rendered/manifest.yaml

```diff
@@ -424,50 +424,14 @@ stringData:
         streams:
         - data_stream:
             dataset: kubernetes.container_logs
+            type: logs
           id: kubernetes-container-logs-${kubernetes.pod.name}-${kubernetes.container.id}
           parsers:
           - container:
               format: auto
               stream: all
           paths:
           - /var/log/containers/*${kubernetes.container.id}.log
-          processors:
```

Comment:
Was these sections removed because it caused the events to not match the integration?